src/Microsoft.AspNetCore.OData/Formatter/ODataInputFormatter.cs

```diff
@@ -140,8 +140,11 @@ public override async Task<InputFormatterResult> ReadRequestBodyAsync(InputForma
                     {
                         throw ex;
                     }
-
```

Comment:
Restore this line. Across the repo, the practice is to leave an empty line after the closing brace, unless it's followed by another closing brace